src/components/ChatEntry.js

```diff
@@ -1,22 +1,30 @@
-import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
 
 const ChatEntry = (props) => {
+  const buttonContent = props.liked ? '❤️' : '🤍';
+
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={props.sender === 'Estragon' ? 'chat-entry remote' : 'chat-entry local'}>
```

Comment:
We could calculate the local or remote class earlier in the component (as with the button emoji) and then insert the result here.
```js
    // earlier in the code (assumes localSender is calculated and passed in)
    const location = (props.sender === props.localSender) ? 'local' : 'remote';

    // here in the JSX
    <div className={`chat-entry ${location}`}>
```